ui/shared/RevisionList.jsx

```diff
@@ -20,7 +52,12 @@ export class RevisionList extends React.PureComponent {
     return (
       <Col className={`${widthClass} mb-3`}>
         {revisions.map((revision) => (
-          <Revision revision={revision} repo={repo} key={revision.revision} />
+          <Revision
+            revision={revision}
+            repo={repo}
+            key={revision.revision}
+            revisionComments={this.state.revisionComments}
```

Comment:
instead of calling `this.state.revisionComments` we should use destructuring from `this.state` like we do for `this.props`